x/blob/README.md

```diff
@@ -2,15 +2,16 @@
 
 ## Abstract
 
-The `x/blob` module enables users to pay for arbitrary data to be published to the Celestia blockchain. Users create a single `MsgWirePayForBlob` transaction that is composed of:
+The `x/blob` module enables users to pay for arbitrary data to be published to the Celestia blockchain. Users create a single `BlobTx` that is composed of:
 
 1. `Blob` (Binary Large OBject): the data they wish to publish
 2. `NamespaceId`: the namespace they wish to publish to
 3. `ShareCommitment`: a signature and a commitment over their data when encoded into shares
+4. `MsgPayForBlob`: a sdk.Msg that pays for the inclusion of the blob.
 
-After the `MsgWirePayForBlob` transaction is submitted to the network, a block producer malleates their transaction into a `MsgPayForBlob` which doesn't include their data (a.k.a blob). Both components get included in the data square in different namespaces: the `MsgPayForBlob` gets included in the transaction namespace and the associated blob gets included in the namespace the user specified in the original `MsgWirePayForBlob`. Further reading: [Message Block Layout](https://github.com/celestiaorg/celestia-specs/blob/master/src/rationale/message_block_layout.md)
+After the `BlobTx` is submitted to the network, a block producer separates their transaction into a `MsgPayForBlob` which doesn't include their data (a.k.a blob). Both components get included in the data square in different namespaces: the `MsgPayForBlob` gets included in the transaction namespace and the associated blob gets included in the namespace the user specified in the original `BlobTx`. Further reading: [Message Block Layout](https://github.com/celestiaorg/celestia-specs/blob/master/src/rationale/message_block_layout.md)
 
-After a block has been created, the user can verify that their data was included in a block via a blob inclusion proof. A blob inclusion proof uses the `ShareCommitment` in the original `MsgWirePayForBlob` and subtree roots of the block's data square to prove to the user that the shares that compose their original data do in fact exist in a particular block.
+After a block has been created, the user can verify that their data was included in a block via a blob inclusion proof. A blob inclusion proof uses the `ShareCommitment` in the original `MsgPayForBlob` and subtree roots of the block's data square to prove to the user that the shares that compose their original data do in fact exist in a particular block.
```

Comment:
> since the share commitment isn't in the MsgPayForBlob or MsgPayForBlobTx

isn't the share commitment in the MsgPayForBlob tho? 

we can get rid of the word "original" tho

Reply:
Sorry I confused myself. You're right b/c the `MsgPayForBlob` contains the share commitment so this was correct as-is 🤦‍♂️